docs/index.md

```diff
@@ -0,0 +1,64 @@
+# Index
+
+-   [Blogs](https://github.com/spiffe/tornjak/blob/main/docs/blogs.md)
+
+-   [Debugging, Hints and Tips for Solving Common Problems with Tornjak](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-hints.md)
+    - [Tornjak Deployment](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-hints.md#tornjak-deployment)
+    - [Tornjak Configuration](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-hints.md#tornjak-configuration)
+    - [User Management](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-hints.md#user-management)
+
+-   [JSON Format for New Entry](https://github.com/spiffe/tornjak/blob/main/docs/newEntry-json-format.md)
+
+-   [Server plugin: Authentication "Keycloak"](https://github.com/spiffe/tornjak/blob/main/docs/plugin_server_authentication_keycloak.md)
+
+-   [Server plugin: Authorization "RBAC"](https://github.com/spiffe/tornjak/blob/main/docs/plugin_server_authorization_rbac.md)
+
+-   [Server plugin: Datastore "SQL"](https://github.com/spiffe/tornjak/blob/main/docs/plugin_server_datastore_sql.md)
+
+-   [Server plugin: SPIRECRDManager](https://github.com/spiffe/tornjak/blob/main/docs/plugin_server_spirecrd.md)
```

Comment:
perhaps these can be placed in a folder - `docs/plugins/server/*`

Reply:
although making this change will require you to go through the documentation and try to find wherever these links are referenced, then update them